zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
Well, I saw this resource,
https://github.com/quantopian/zipline/wiki/Development-Environment.

But there is nothing about developing on windows.  I guess that's to be
expected.


On Tue, Nov 3, 2015 at 7:53 AM, Aaron Elmquist <elmq0022@umn.edu> wrote:

> Yes that seems right too me. I coded something up to that effect this
> morning.
>
> I couldn't quite get the test to run without throwing an error.
>
> I will have to take a look tomorrow and see if I can track down the
> issue.
>
> By the way, is there a guide on setting up an environment for developing
> zipline?
>